packages/reactivity/src/computed.ts

```diff
@@ -47,11 +47,11 @@ export class ComputedRefImpl<T = any> implements Subscriber {
   /**
    * @internal
    */
-  readonly dep = new Dep(this)
+  readonly dep = new Dep(this);
```

Comment:
Please format the code

Reply:
emm, Is there no automatic formatting configuration? `eslint --fix` doesn't work.

Reply:
We use Prettier for code style.

Reply:
And, `pnpm format` and `format-check` did NOT make any modifications.

Reply:
I think it's a bug of Prettier, let's ignore it for Vue.

https://prettier.io/playground/#N4Igxg9gdgLgprEAucAbAhgZ0wAgII7AA6UOOATnOgCbSoCeO1cADjgLw5RwDuOAIqwAUMABYBLTAEoSZSjTqMA2gCUqYGOIBucAGIYA5pgB0ASQDKAfRUBRXQF0OOGOQCucANwkAviAA0IBAsmtCYyKDo5OQQPAAKkQhhKOioPOj0YQEARuToYADWcDDm6AC2cAAy4tzIAGYpmHDZuQVF5ix51QbILu4BjaXidQ1NIHAAHixw5OLlsCkAKtNQkeJwSfWojf1dqHAAiq4Q8MNbowBWmOPmuwdHJ0ib2yAAjvdwsdEsSSBYALTcODMaj+EAudDiVBdADCEFKpXQyF+qFQoMwtzwMBc4iyrngsWmVRqjxGAVEMFKqAA6hJ4JgOmA4OZEuJNFpWfQkWBsKCtO5TFBmLBzGAZsE8ILzDB6HtTs8WNFGlTciwkQr1tMdKDqo1yDBPugDAi5aMOuRdUjpVNMKLxMFQQrqjAqeJqGJkAAOAAMAUob3ElANRsRJLOARg6CyLrdomQACYAq5GgtIxtSWNSlkgcCKugoAZXIa9BByAisV0keg8RAQN5vEA